rn/Teacher/src/modules/submissions/list/__tests__/map-state-to-props.test.js

```diff
@@ -37,12 +37,12 @@ let t = {
 function createHappyPathTestState () {
   const enrollments: EnrollmentsState = {
     '0': t.enrollment({ id: '0', user_id: '0', type: 'TeacherEnrollment' }),
-    '1': t.enrollment({ id: '1', user_id: '1', user: t.user({ id: '1', name: 'S1', sortable_name: 'S1' }) }),
-    '2': t.enrollment({ id: '2', user_id: '2', user: t.user({ id: '2', name: 'S2', sortable_name: 'S2' }) }),
-    '3': t.enrollment({ id: '3', user_id: '3', user: t.user({ id: '3', name: 'S3', sortable_name: 'S3' }) }),
-    '4': t.enrollment({ id: '4', user_id: '4', user: t.user({ id: '4', name: 'S4', sortable_name: 'S4' }) }),
+    '1': t.enrollment({ id: '1', user_id: '1', user: t.user({ id: '1', name: 'Student1', sortable_name: 'S1' }) }),
+    '2': t.enrollment({ id: '2', user_id: '2', user: t.user({ id: '2', name: 'Student2', sortable_name: 'S2' }) }),
+    '3': t.enrollment({ id: '3', user_id: '3', user: t.user({ id: '3', name: 'Student3', sortable_name: 'S3' }) }),
+    '4': t.enrollment({ id: '4', user_id: '4', user: t.user({ id: '4', name: 'Student4', sortable_name: 'S4' }) }),
     // user in two sections or as a TA & Student
-    '5': t.enrollment({ id: '5', user_id: '4', user: t.user({ id: '4', name: 'S4', sortable_name: 'S5' }) }),
+    '5': t.enrollment({ id: '5', user_id: '4', user: t.user({ id: '4', name: 'Student4', sortable_name: 'S5' }) }),
```

Comment:
Changed these to ensure that the tests were getting the sortable_name and not the name.